test/e2e/tests/setup.ts

```diff
@@ -28,7 +23,8 @@ export const setup = async (): Promise<SetupResult> => {
 
   const publicClient = setupPublicClient();
   const engine = setupEngine();
-  const backendWallet = await getEngineBackendWallet(engine);
+  // const backendWallet = await getEngineBackendWallet(engine);
+  const backendWallet = "0x50495dacda9553e50dad15cd3a3016743f71b4c9";
```

Comment:
Did you mean for this change?

Reply:
nope, reverted. This one keeps falling through lol, high time to make it configurable properly.